app/assets/stylesheets/_variables.scss

```diff
@@ -16,20 +16,54 @@ $small-spacing: $base-spacing / 2;
 $base-z-index: 0;
 
 // Colors
-$blue: #477dca;
-$dark-gray: #333;
-$medium-gray: #999;
-$light-gray: #ddd;
-
-// Font Colors
-$base-background-color: #fff;
-$base-font-color: $dark-gray;
-$action-color: $blue;
-
-// Border
-$base-border-color: $light-gray;
+$dark-color: #333;
+$light-color: #fff;
+
+// Background
+$base-background: #fff;
```

Comment:
The idea here was to try to be consistent in using `color` for text colors and `background` for background-colors. But it's not a big deal. And it might not be that consistent either. Fixing this.